tests/nnstreamer_edge/query/unittest_query.cc

```diff
@@ -266,6 +208,244 @@ TEST (tensorQuery, clientAlone_n)
   g_free (pipeline);
 }
 
+
+/**
+ * @brief Test for tensor_query_server custom coonnection
```

Comment:
```suggestion
 * @brief Test for tensor_query_server custom connection
```

Hmm,, Why didn't the typo action detect it 🤔?

Reply:
It's bug..😥
I tested it on my personal github action, it's not detected.

Changes with typos: coonnection Finallizes
```
diff --git a/tests/nnstreamer_edge/query/unittest_query.cc b/tests/nnstreamer_edge/query/unittest_query.cc
index 08641068..eaa9d4b8 100644
--- a/tests/nnstreamer_edge/query/unittest_query.cc
+++ b/tests/nnstreamer_edge/query/unittest_query.cc
@@ -210,7 +210,7 @@ TEST (tensorQuery, clientAlone_n)
 
 
 /**
- * @brief Test for tensor_query_server custom coonnection
+ * @brief Test for tensor_query_server custom coonnection Finallizes
  */
 TEST (tensorQuery, serverCustomNormal)
 {
```
Detected result:
```
./typos
jq: jq-1.6
$ ./typos . --config ./.github/workflows/_typos.toml
Warning: "Finallizes" should be "Finalizes".
error: `Finallizes` should be `Finalizes`
  --> ./tests/nnstreamer_edge/query/unittest_query.cc:213:59
    |
213 |  * @brief Test for tensor_query_server custom coonnection Finallizes
    |                                                           ^^^^^^^^^^
    |
Error: Process completed with exit code 2.
```

I request to fix it. https://github.com/crate-ci/typos/issues/1090